### PR TITLE
[HB-4523] Updating CHANGELOG.md for 3.1.0 Release

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**
# Description

Updating CHANGELOG.md in repo & package.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file using the standards as defined at [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0).
 
+### Version 3.1.0 *(2022-09-08)*
```
Comment (cb-jpadilla) [inline CHANGELOG.md:4]: I think the release is set for the 15, is it not?

File: com.chartboost.helium/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
 # Changelog
 All notable changes to this project will be documented in this file using the standards as defined at [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0).
 
+### Version 3.1.0 *(2022-09-08)*
```
Comment (cb-jpadilla) [inline com.chartboost.helium/CHANGELOG.md:4]: Same here. Don't think release is until the 15th.

File: CHANGELOG.md (hunk 1 above)
Comment (SCastanedaMunoz) [reply CHANGELOG.md:4]: Oh well, I was mostly indicating when it was being updated, not the release date. But it makes sense for it to be the release date. 👍🏻

Comment (SCastanedaMunoz) [reply CHANGELOG.md]: I will keep that in mind for the future.

**Review by cb-jpadilla — APPROVED** [review]
lgtm!!

**Pull request merged**